File: app/src/main/java/com/neome/api/meta/base/dto/DefnMapOfDynamicRule.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdFieldDynamicRule;

import java.util.Map;

@SuppressWarnings("unused")
public class DefnMapOfDynamicRule
{
  public MetaIdFieldDynamicRule[] keys;

  public Map<MetaIdFieldDynamicRule, DefnDtoDynamicRule> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnMapOfDynamicRule.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnDtoDynamicRule
import com.neome.api.meta.base.Types.MetaIdFieldDynamicRule

interface DefnMapOfDynamicRule
{
  val keys: List<MetaIdFieldDynamicRule>
  val map: Map<MetaIdFieldDynamicRule, DefnDtoDynamicRule>
}